.utility/update-docs-repo.sh

```diff
@@ -1,28 +1,29 @@
-#!/bin/sh
+#!/bin/bash
 
-# dev.loklak.org Update trigger script
+cp README.rst $HOME/.
 
-# This script is used to trigger the update process
-# of dev.loklak.org
+git config --global user.email "travis@travis-ci.org"
+git config --global user.name "travis-ci"
 
-WORK="$HOME/central-docs"
-REPO="loklak/dev.loklak.org"
-PRIVATE_KEY="$(pwd)/.utility/loklakserver.2"
+git subtree --prefix=docs/ split -b documentation
 
-echo "Preparing ssh agent ..."
-eval $(ssh-agent -s)
-ssh-add $PRIVATE_KEY
+cd $HOME
 
-echo "Preparing git ..."
-git config --global user.email "travis@travis-ci.org"
-git config --global user.name "travis-ci"
+git clone --quiet --branch=gh-pages git@github.com:loklak/dev.loklak.org.git gh-pages
```

Comment:
Don't clone gh-pages branch. It will contain the built documentation. Clone master instead